server/issue.go

```diff
@@ -778,16 +778,47 @@ func (s *Server) getPipelineCreateForDatabaseSchemaAndDataUpdate(ctx context.Con
 			return envKeys[i].order < envKeys[j].order
 		})
 		for _, env := range envKeys {
+			taskList, taskIndexDagList := getTaskAndDagListByVersion(envToDatabaseMap[env])
 			create.StageList = append(create.StageList, api.StageCreate{
-				Name:          env.name,
-				EnvironmentID: env.id,
-				TaskList:      envToDatabaseMap[env],
+				Name:             env.name,
+				EnvironmentID:    env.id,
+				TaskList:         taskList,
+				TaskIndexDAGList: taskIndexDagList,
 			})
 		}
 	}
 	return create, nil
 }
 
+// getTaskAndDagListByVersion adds the task dependencies for tasks belonging to the same database.
```

Comment:
should mention the version order (ASC or DESC)

Reply:
Done.